docs/modules/user-guide/nav.adoc

```diff
@@ -1,32 +1,36 @@
 * xref:index.adoc[]
-* xref:devfile-registry.adoc[]
 
-* xref:authoring-stacks.adoc[]
-** xref:adding-schema-version-to-a-devfile.adoc[]
-** xref:adding-a-name-to-a-devfile.adoc[]
-** xref:adding-projects-to-a-devfile.adoc[]
-** xref:adding-commands-to-a-devfile.adoc[]
-** xref:adding-components-to-a-devfile.adoc[]
-*** xref:adding-kubernetes-component-to-a-devfile.adoc[]
-*** xref:adding-container-component-to-a-devfile.adoc[]
-*** xref:specifying-persistent-storage.adoc[]
-*** xref:limiting-resources-usage.adoc[]
-*** xref:defining-environment-variables.adoc[]
-*** xref:defining-endpoints.adoc[]
-*** xref:defining-kubernetes-resources.adoc[]
+* Developer
+** xref:using-stacks.adoc[]
```

Comment:
Added a `Using stacks` section before `Authoring stacks` to help the user workflow. Thoughts? 

Also, I'm not sure why the `Using stacks` isn't rendering properly.